src/database/mockdata/articledata.js

```diff
@@ -0,0 +1,13 @@
+export const articleSample = {
```

Comment:
Please ensure this file doesn't show as an addition to the codebase since they have nothing related to this PR.

Reply:
Thank you, this has no connection to the database but has been resolved locally once I pushed it will take effect.